components/channel_info_modal/channel_info_modal.tsx

```diff
@@ -90,6 +106,17 @@ export default class ChannelInfoModal extends React.PureComponent {
                 purpose: notFound,
                 header: notFound,
                 id: notFound,
+                team_id: notFound,
```

Comment:
Interesting. I didn't know we passed around a fake channel object here. I feel like that could cause other issues, but I'm not too concerned about that right now. Nothing else needs to change here, but I thought I'd mention it.